src/main/java/com/obj/nc/flows/dataSources/properties/jdbc/JdbcJobProperties.java

```diff
@@ -37,5 +37,6 @@
 
     @NotEmpty
     private String cron;
-    
+
+    private String externalIdKey;
```

Comment:
Is this name of the colum from the DS that is considered to be PK? If so let's call it externalIDComunName

Reply:
renamed